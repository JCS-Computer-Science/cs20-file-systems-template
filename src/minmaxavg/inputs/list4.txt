37.6
95.4
98.6
67.7
14.1
98.5
36.4
35.5
83.8
16.7
17.7
66.4
37.6
17
37.6
20.3
39.7
7
80
91.9
58.8
53.3
78.4
28.1
68.7
71.9
88.7
60.2
72.3
97.3
59.8
25.4
16.9
5.6
89.2
18.5
33.9
55.2
16.9
19
93.9
66.5
38.3
56.4
99.3
22.4
94.3
49
92
66.3
69.6
47.7
77
42.7
69.2
35.7
66.4
70.9
49.2
23.6
55.9
49.7
31.2
35.6
27.5
65.8
53.8
0.1
4
61.6
64
81.8
92.9
3.8
52.9
4.6
78.4
70.5
3.2
13.9
88.3
79.7
91
28.7
50.3
94
67.1
70.5
96.1
42.4
54.7
36.9
6.3
74.3
69
55.8
6.1
