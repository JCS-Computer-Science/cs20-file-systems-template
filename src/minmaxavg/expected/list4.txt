Min: 0 Max: 99.3 Avg: 52.15